__tests__/integrations/BingAds/browser.test.js

```diff
@@ -0,0 +1,148 @@
+import { BingAds } from '../../../src/integrations/BingAds';
+import { event, query, products } from './__mocks__/data';
+
+describe('BingAds init tests', () => {
+  let Bingads;
+  test('Testing init call of Bing ads', () => {
+    Bingads = new BingAds({ tagID: '12567839' }, { loglevel: 'debug' });
+    Bingads.init();
+    expect(typeof window.bing12567839).toBe('object');
+  });
+});
+
+describe('BingAds page', () => {
+  let bingAds;
+  const output = [];
+  beforeEach(() => {
+    bingAds = new BingAds({ tagID: '12567839' }, { loglevel: 'debug' });
+    bingAds.init();
+    window.bing12567839.push = jest.fn((x) => output.push(x));
+  });
+
+  test('send pageview', () => {
+    bingAds.page({
+      context: {},
+      properties: {
+        category: 'test category',
+        path: '/test',
+        url: 'http://localhost',
+        referrer: '',
+        title: 'test page',
+        testDimension: 'abc',
+      },
+    });
+    expect(output[0]).toEqual('pageLoad');
```

Comment:
Can we assert other fields as well?

Reply:
Only passing `pageLoad` in Page Call. No other field are supported.